charts/camunda-platform/templates/NOTES.txt

```diff
@@ -78,10 +86,10 @@ Default user and password: "demo/demo"
 As part of Identity an ingress definition can be deployed, but you require to have an Ingress Contoller for that Ingress to be Exposed.
 In order to deploy the ingress manifest, set `identity.ingress.enabled` to `true`.
 
-If you don't have an ingress controller you can use kubectl port-forward to access identity from outside the cluster:
+If you don't have an ingress controller you can use kubectl port-forward to access Identity from outside the cluster:
 
 > kubectl port-forward svc/{{ .Release.Name }}-identity 8080:80
-> kubectl port-forward svc/{{ .Release.Name }}-keycl 18080:80
+> kubectl port-forward svc/{{ include "common.names.dependency.fullname" (dict "chartName" "keycloak" "chartValues" .Values.identity.keycloak "context" $) | trunc 20 | trimSuffix "-" }} 18080:80
```

Comment:
Keycloak has the weird property to cap the names at 20 chars, the reason is that wildfly only supports a certain amount of characters :shrug: :laughing: